### PR TITLE
Moabs v1.3.7.5

### PR DESCRIPTION
**Pull request by lijinbio**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/cmsip/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: "87471ff0a8ce40a10bed29afef1ce2f691f79f905d91b02088ee95c9ddf43c45"
 
 build:
-  number: 0
+  number: 1
```
Comment (npavlovikj) [inline recipes/cmsip/meta.yaml]: Sorry for my previous comment - I didn't realize you had 2 different recipes here. Is there a reason why you are re-building "cmsip"? Looks like the recipe was updated 3 ago.

Comment (lijinbio) [reply recipes/cmsip/meta.yaml]: @npavlovikj Thank you for your comments. No, "cmsip" should not rebuild. It's my mistake to mis-edit its recipe. This PR is for "moabs". I have updated the PR. Thank you for your review.

Comment (lijinbio) [reply recipes/cmsip/meta.yaml]: @npavlovikj Hi Natasha, my updated PR has passed the Circle CI build. Can you help review my PR? Thank you for your help.

**Comment by npavlovikj** [comment]
Thanks for the update and clarification @lijinbio - your recipe is now merged!

**Pull request merged**